src/seedu/addressbook/data/tag/TaggingList.java

```diff
@@ -0,0 +1,31 @@
+package seedu.addressbook.data.tag;
+
+import java.util.ArrayList;
+
```

Comment:
Please include a header comment to indicate what this interface/class/method does?